src/Uno.UI/UI/Xaml/Window/Window.cs

```diff
@@ -324,6 +327,20 @@ UIElement value
 	{
 	}
 
+	public bool ExtendsContentIntoTitleBar
+	{
+		get => _extendsContentIntoTitleBar;
+		set
+		{
+			if (_extendsContentIntoTitleBar != value)
+			{
+				_extendsContentIntoTitleBar = value;
+				ExtendsContentIntoTitleBarChanged?.Invoke(value);
```

Comment:
I think with the current implementation the setting may be only applied when the window is already shown, but we should also support the scenario where it is set before `Activate()` call. It is probably just about force-applying it right after the window is shown I presume

Reply:
As you noticed, this already works since the "native window" is created during the construction of the WinUIWindow, so any changes to ExtendsContentIntoTitleBar will happen after the subscription is is place.